docs/integrations/pack-docs.md

```diff
@@ -55,3 +73,195 @@ It is possible to add an image placeholder which links to an external video. For
 [![Video Name](https://img.youtube.com/vi/[YOUTUBE_VIDEO_ID]/0.jpg)](https://www.youtube.com/watch?v=[YOUTUBE_VIDEO_ID] "Video Name")
 ```
 An example of including a YouTube video can be seen at the [Zoom Feed README](https://github.com/demisto/content/blob/master/Packs/FeedZoom/README.md).
+
+## Pack Keywords, Tags, Use-cases & Categories
+In order classify packs and make them easier to find, there are a number of pack metadata elements that can be used:
+- Use-case -- a name of a situation in which packs can be used - usually the pack name. 
+
+ Examples: Impossible Traveler, PCAP Analysis
+
+- Tags -- words from the same semantic field of the use-case - they help find packs using filters or the search bar, and are visible on the screen to help understand what the pack is all about.
+
+ Examples: GDPR, PII, TIM
+
+- Keywords -- like tags, but these don’t show in the UI.
```

Comment:
I would elaborate this a little more: I think it should mention that keywords are not used in the UI itself, but they are important when searching the pack in the UI

Reply:
This is mentioned in the tags right before, then keywords is referencing to tags as the only difference is whether they show in the UI
![image](https://user-images.githubusercontent.com/43602124/91473907-3634d700-e8a2-11ea-8abe-5a02d8563d66.png)

If you feel this is insufficient or unclear I can improve it

Reply:
Yes, I suggest to change it a bit because someone might look directly at Keywords and don't look at the lines above :)